packages/server/src/environment.ts

```diff
@@ -126,20 +127,26 @@ const environment = {
   getDefaults: () => {
     return DEFAULTS
   },
+  ISOLATEDVM_QUERY_TRANSFORMERS: !!process.env.ISOLATEDVM_QUERY_TRANSFORMERS,
```

Comment:
can we keep the environment variables together above the final exports/functions in the `environment.ts` files? Makes it easier to not miss some environment variables in the list!